packages/freezeframe/src/index.ts

```diff
@@ -124,8 +124,12 @@ class Freezeframe {
   private _process(freeze: Freeze): Promise<Freeze> {
     return new Promise((resolve) => {
       const { $canvas, $image, $container } = freeze;
-      const { clientWidth, clientHeight } = $image;
+      const { width, height } = $image.getClientRects()[0];
```

Comment:
new way to read the computed dimensions of the image that supports floating point numbers.